sample-lnd.conf

```diff
@@ -356,6 +492,35 @@ litecoin.node=ltcd
 ; in with lnd's traffic.
 ; tor.streamisolation=true
 
+
+; Enable Tor stream isolation by randomizing user credentials for each
+; connection.
+; tor.streamisolation=1                                   
+
+; The host:port that Tor is listening on for Tor control connections (default:
+; localhost:9051)
+; tor.control=localhost:9091                                      
+
+; IP address that Tor should use as the target of the hidden service
+; tor.targetipaddress=                                  
+ 
+; The password used to arrive at the HashedControlPassword for the control port.
+; If provided, the HASHEDPASSWORD authentication method will be used instead of
+; the SAFECOOKIE one.
+; tor.password=plsdonthackme                                         
+ 
+; Automatically set up a v2 onion service to listen for inbound connections
+; tor.v2=1                                                
+ 
+; Automatically set up a v3 onion service to listen for inbound connections
+; tor.v3=1                                               
```

Comment:
I think we should use `true` or `false` for boolean flags everywhere to make the config more consistent. At least there was a PR a while ago that did it for the then existing values.

Reply:
Updated to be consistent everywhere!